WordPress/src/main/java/org/wordpress/android/ui/accounts/BlogUtils.java

```diff
@@ -188,4 +189,22 @@ public static void convertToLowercase(Editable s) {
             s.replace(0, s.length(), lowerCase);
         }
     }
+
+    @NonNull
+    public static Set<String> planTags() {
+        String[] extraFields = {"plan_product_id"};
+        List<Map<String, Object>> blogs = WordPress.wpDB.getBlogsBy("dotcomFlag=1", extraFields);
+        Set<String> tags = new HashSet<>();
+
+        for (Map<String, Object> blog : blogs) {
+            Integer planId = MapUtils.getMapInt(blog, "plan_product_id");
+            if (planId == 0) {
```

Comment:
Let's add a small comment here about the special meaning of the `0` value. WDYT?